modules/cli/src/main/scala/scala/cli/internal/ProcUtil.scala

```diff
@@ -1,5 +1,7 @@
 package scala.cli.internal
 
+import java.io.InputStream
+import java.net.{HttpURLConnection, URL}
```

Comment:
With the suggestion below, that should become 
```suggestion
import java.net.URL
```